packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -463,7 +478,9 @@ async function createReconciliationReport(params) {
   const filesInCumulus = {
     okCount: 0,
     onlyInS3: [],
-    onlyInDynamoDb: []
+    onlyInDynamoDb: [],
+    S3Errors: [],
+    ESErrors: []
```

Comment:
These didn't get rolled back.   Because we have a pretty random set of possible values for the Granule.Files[Objects], I started collecting objects that couldn't be parsed as files, keys. anyway, that's for a different PR. I will pull this out.